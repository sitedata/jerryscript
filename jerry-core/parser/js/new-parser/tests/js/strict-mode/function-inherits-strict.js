/* This test contains strict mode test, whether function inherits strict mode. */

"use strict";

function inherits_strict() {
  with ( obj ) next_statement;
}
