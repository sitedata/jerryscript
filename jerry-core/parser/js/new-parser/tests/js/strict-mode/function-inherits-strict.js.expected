
--- Script parsing start ---

  Note: switch to strict mode

  [  1] CBC_PUSH_LITERAL idx:0->string(use strict)
  [  0] CBC_POP_BLOCK

--- Function parsing start ---


--- Script parsing end ---

Parse error 'With statement not allowed in strict mode.' at line: 6 col: 3
